### PR TITLE
log load info, on worker clusters

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: atomic_reactor/plugins/build_orchestrate_build.py [hunk 1]
```diff
@@ -186,6 +189,8 @@ def choose_cluster(self, platform):
         selected = min(clusters, key=lambda c: c.load)
         self.log.info('platform %s will use cluster %s',
```
Comment (vrutkovs) [inline atomic_reactor/plugins/build_orchestrate_build.py]: Remove this line?

File: atomic_reactor/plugins/build_orchestrate_build.py [hunk 2]
```diff
@@ -186,6 +189,8 @@ def choose_cluster(self, platform):
...
                       platform, selected.cluster.name)
+        self.log.debug('platform %s will use cluster %s with the lowest load %s',
+                       platform, selected.cluster.name, selected)
```
Comment (vrutkovs) [inline atomic_reactor/plugins/build_orchestrate_build.py]: `selected` here is a cluster name, it should be `selected.load`

File: atomic_reactor/plugins/build_orchestrate_build.py (hunk 1 above)
Comment (vrutkovs) [inline atomic_reactor/plugins/build_orchestrate_build.py]: Lets use one message for picked cluster and its load

File: atomic_reactor/plugins/build_orchestrate_build.py [hunk 3]
```diff
@@ -172,6 +172,9 @@ def get_cluster_info(self, cluster, platform):
         osbs = OSBS(conf, conf)
         current_builds = self.get_current_builds(osbs)
         load = current_builds / cluster.max_concurrent_builds
+        self.log.debug('enabled cluster %s for platform %s has load %s and active builds %s and '
+                       'max concurrent builds %s', cluster.name, platform, load, current_builds,
```
Comment (twaugh) [inline atomic_reactor/plugins/build_orchestrate_build.py]: How about `active builds %s/%s`?

File: atomic_reactor/plugins/build_orchestrate_build.py (hunk 2 above)
Comment (twaugh) [inline atomic_reactor/plugins/build_orchestrate_build.py]: I don't really think we need this debug line if we have the get_cluster_info() one.

File: atomic_reactor/plugins/build_orchestrate_build.py (hunk 3 above)
Comment (rcerven) [inline atomic_reactor/plugins/build_orchestrate_build.py]: ok

**Pull request merged**